packages/jest-config/src/normalize.js

```diff
@@ -367,6 +367,28 @@ export default function normalize(options: InitialOptions, argv: Argv) {
     ),
   );
 
+  if (!options.setupFilesAfterEnv) {
+    options.setupFilesAfterEnv = [];
+  }
+
+  if (
+    options.setupTestFrameworkScriptFile &&
+    options.setupFilesAfterEnv.length >= 1
```

Comment:
`options.setupFilesAfterEnv` is defined. We need to know if it's being used.

Reply:
`>0` might be more legible? not sure